recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -2,6 +2,8 @@ EXTRA_OEMESON:append:tegra = " -Dglvnd=true"
 DEPENDS:append:tegra = " libglvnd"
 PROVIDES:tegra = "virtual/mesa virtual/libgbm"
 
+PACKAGECONFIG:append:tegra = "gbm"
+
```

Comment:
`gbm` is in the PACKAGECONFIG by default, so this shouldn't be needed.  If it _is_ needed in some cases, make sure you prefix it with a blank